pkg/server/serverpb/admin.proto

```diff
@@ -141,6 +141,11 @@ message TableDetailsResponse {
 
   // range_count is the size of the table in ranges. This provides a rough
   // estimate of the storage requirements for the table.
+  // TODO(mrtracy): The TableStats method also returns a range_count field which
```

Comment:
Since that is an implementation detail and not an interface detail, I added a TODO in the implementation of TableStats (admin.go)